.github/workflows/visual-regression-tests.yml

```
@@ -0,0 +1,40 @@
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request:
+    types:
+      - ready_for_review
+
+jobs:
+  visual-regression-test-components:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "14"
+      - name: Check for cached Ruby gems
+        uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: ${{ runner.os }}-gems-
```

```suggestion
          path: vendor/bundle
          key: bundle-${{ hashFiles('**/Gemfile.lock') }}
          restore-keys: bundle
```

This suggestion brings it in line with https://docs.publishing.service.gov.uk/manual/test-and-build-a-project-with-github-actions.html#simple-ruby-application - we don't gain anything by specifing the os given we're only building on one.

---

Ah, I meant to remove this - thank you. I initially added this OS differentiator as I thought that might give us the ability to take screenshots on Windows, Linux, and macOS - but then discovered that's not how Percy works.